tools/probers/bazelrbe/bazelrbe.go

```diff
@@ -125,9 +125,7 @@ func runProbe() error {
 
 	args := []string{"//" + *proberName + ":all"}
 	if *bazelArgs != "" {
-		for _, extraArg := range strings.Split(*bazelArgs, " ") {
-			args = append(args, extraArg)
-		}
+		args = append(args, strings.Split(*bazelArgs, " ")...)
```

Comment:
nit: this feels slightly more readable -

```suggestion
		extraArgs := strings.Split(*bazelArgs, " ")
		args = append(args, extraArgs...)
```